### ISSUE TITLE
Remove most of qml7 xyz files

### ISSUE THREAD

**Issue by larsbratholm** [qmlcode/qml#63]
There is no reason to have more than a few. If many is needed for a test we can include a pickle with coordinates / nuclear charges

**Comment by andersx** [comment]
I agree. There are separate tests for the xyz parsing, so would not interfere with anything. Could use ASE or a datacontainer class
